src/options/quantifiers_options.toml

```
@@ -1318,6 +1318,13 @@ name   = "Quantifiers"
   default    = "3"
   help       = "maximum number of instantiation rounds for sygus verification calls (-1 == no limit, default is 3)"
 
+[[option]]
+  name       = "sygusVerifyTimeout"
+  category   = "regular"
+  long       = "sygus-verify-timeout=N"
+  type       = "int64_t"
```

```suggestion
  type       = "uint64_t"
  default   = 0
```
the time limit is a `uint64_t` internally. Also, to make this explicit and to easily show it in the documentation, specifying defaults have recently become mandatory.